### ISSUE TITLE
Could you provide the R-50 backbone file?

### ISSUE THREAD

**Issue by qjy981010** [sabarim/STEm-Seg#2]
Hello, thanks for your good work. I'm trying to train your model with ResNet-50 but there is no pre-trained ResNet-50 backbone file. Could you provide the R-50 backbone file for us?

**Comment by Ali2500** [comment]
Hi,

I've added the ResNet-50 pretrained backbone from Mask-RCNN to the model zoo:
https://omnomnom.vision.rwth-aachen.de/data/STEm-Seg/models/mask_rcnn_R_50_FPN_backbone.pth

I haven't really tested this, but I think it should work if you just make the following changes:
* Change `cfg.MODEL.BACKBONE.TYPE` to "R-50-FPN"
* Change `cfg.MODEL.BACKBONE.PRETRAINED_WEIGHT` to "mask_rcnn_R_50_FPN_backbone.pth"

Let me know if you run into any issues.
